example/ck_tile/35_batched_transpose/batched_transpose_api.cpp

```diff
@@ -1,28 +1,26 @@
 // SPDX-License-Identifier: MIT
 // Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
-#include "batched_transpose_example.hpp"
+#include "batched_transpose.hpp"
 #include <iostream>
 
 template <typename ts_type,
-          ck_tile::index_t block_x,
-          ck_tile::index_t block_y,
-          ck_tile::index_t warp_x,
-          ck_tile::index_t warp_y,
-          ck_tile::index_t thread_x,
-          ck_tile::index_t thread_y>
+          ck_tile::index_t block_w,
+          ck_tile::index_t block_h,
+          ck_tile::index_t warp_w,
+          ck_tile::index_t warp_h,
+          ck_tile::index_t thread_w,
+          ck_tile::index_t thread_h>
 float batched_transpose_dispatch(batched_transpose_kargs& a, ck_tile::stream_config& s)
 {
-    uint32_t dim_block_h = (a.height + block_y - 1) / block_y;
-    uint32_t dim_block_w = (a.width + block_x - 1) / block_x;
-    uint32_t dim_stride  = a.height * a.width;
+    uint32_t dim_stride = a.height * a.width;
 
     a.dim_stride  = dim_stride;
```

Comment:
we should name this `batch_stride` and it should be given by user (in order to accept padded tensors)